students/hyska_monika/lesson_14_decorators/exercise3.py

```diff
@@ -0,0 +1,22 @@
+"""
+Write @logged decorator using logger_wrapper
+from lesson 6. Apply it to several functions
+to demonstrate that it works
+"""
```

Comment:
You forgot about that file probably, below you have the solution to sort i.e. exercise 2.